### ISSUE TITLE
[BUG] Field 'fileName' on type 'StackFrame' is too long. Expected: 1024 characters

### ISSUE THREAD

**Issue by kosmonavtsv** [microsoft/ApplicationInsights-JS#2086]
**Description/Screenshot**

I am attempting to log an exception in response I get a 400 response with the following:
```
{
    "itemsReceived": 1,
    "itemsAccepted": 0,
    "errors": [
        {
            "index": 0,
            "statusCode": 400,
            "message": "100: Field 'fileName' on type 'StackFrame' is too long. Expected: 1024 characters, Actual: 2021"
        }
    ],
}
```
The number of characters in the `fileName` field is really more than 1024, here is the path to this field in the query body : 
`request.data.baseData.exceptions[0].parsedStack[0].fileName`.

**Steps to Reproduce**

 - OS/Browser: any
 - SDK Version: 2.8.13, 1.0.21

**Expected behavior**
Error is logged.

**Additional context**
💡  A similar problem was solved for the assembly field here: https://github.com/microsoft/ApplicationInsights-JS/issues/1990.

**Comment by MSNev** [comment]
Note: v1.0.x will not be updated to address this issue

**Comment by MSNev** [comment]
Now fully released as part of version 2.8.15
